src/python/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+# Mastho - 2020
+link_libraries(kdmp-parser)
+find_package (Python3 COMPONENTS Interpreter Development)
+
+# Python on Windows Debug build only load $_d.pyd and require python_d.exe
+if(CMAKE_BUILD_TYPE STREQUAL "Debug" AND WIN32)
+    set(PYMODULE kdmp_d CACHE INTERNAL "")
```

Comment:
What's the `CACHE INTERNAL ""` part for?

Reply:
PYMODULE is used in another cmake script so need CACHE keyword. INTERNAL probably not required.